app/src/common/constants.ts

```diff
@@ -1,3 +1,21 @@
+import { CoronaMarketsLogo } from '../components/common/logos/corona_markets'
+import { OmenLogo } from '../components/common/logos/omen'
+
+enum Version {
+  GNO = 'gno',
+  CORONA = 'corona',
+  OMEN = 'omen',
+}
+
+export const VERSION = process.env.REACT_APP_VERSION || 'omen'
```

Comment:
Check that the given `REACT_APP_VERSION` is one of the allowed values, and make the type of `VERSION` be `Version`.